### ISSUE TITLE
Export Fails

### ISSUE THREAD

**Issue by ColonelPanicks** [alces-software/userware#1]
```directory> export
Traceback (most recent call last):
  File "src/directory", line 7, in <module>
    directory()
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/opt/directory/cli/src/directory.py", line 29, in invoke
    super().invoke(ctx)
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/directory/cli/src/appliance_cli/sandbox.py", line 27, in sandbox
    allow_internal_commands=False,
  File "/opt/directory/cli/venv/src/click-repl/click_repl/__init__.py", line 194, in repl
    group.invoke(ctx)
  File "/opt/directory/cli/src/directory.py", line 29, in invoke
    super().invoke(ctx)
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/directory/cli/venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/opt/directory/cli/src/import_export.py", line 68, in export
    success_message = _record_export_success_message(export_filepath)
  File "/opt/directory/cli/src/import_export.py", line 126, in _record_export_success_message
    _export_url(export_filename)
  File "/opt/directory/cli/src/import_export.py", line 143, in _export_url
    appliance_url = appliance_cli.config_utils.appliance_url()
  File "/opt/directory/cli/src/appliance_cli/config_utils.py", line 18, in appliance_url
    read_config(CONFIG.CLUSTERWARE_ACCESS_CONFIG)
  File "/opt/directory/cli/src/appliance_cli/utils.py", line 54, in read_config
    for line in open(config_file_path).readlines()
FileNotFoundError: [Errno 2] No such file or directory: '/opt/clusterware/etc/access.rc'
```

Missing config file, what should be in `/opt/clusterware/etc/access.rc` and move that into `/opt/userware/`

**Comment by ColonelPanicks** [comment]
I believe this is not just a missing config file but also the missing web front-end. From the unit testing:
```
[root@infra01 cli]# make test
. venv/bin/activate && pytest src/ --ignore=src/nagios_interface/venv
========================================================= test session starts =========================================================
platform linux -- Python 3.5.5, pytest-3.0.5, py-1.4.32, pluggy-0.4.0
rootdir: /opt/directory/cli, inifile:
plugins: mock-1.5.0
collected 34 items

src/test_import_export.py .F..
src/test_ipa_utils.py ........
src/test_user.py .........
src/appliance_cli/test_command_generation.py ........
src/appliance_cli/test_support.py .....

============================================================== FAILURES ===============================================================
______________________________________ test_import_outputs_last_password_generated_for_each_user ______________________________________

tmpdir = local('/tmp/pytest-of-root/pytest-0/test_import_outputs_last_passw0')
mocker = <pytest_mock.MockFixture object at 0x7f05ee298a20>, monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f05ee298128>

    def test_import_outputs_last_password_generated_for_each_user(
            tmpdir,
            mocker,
            monkeypatch
    ):
        mocker.spy(utils, 'directory_run')

        _mock_ipa_run_output(monkeypatch)

        # TODO this tests a lot of related things; could be worth breaking up into
        # more discrete tests.
        test_record = tmpdir.mkdir('somedir').join('record').strpath
        with open(test_record, 'w') as record_file:
            record_file.writelines([
                'user create newuser --first new --last user\n',

                'user create anotheruser --first someone --last else\n',
                'user modify anotheruser --new-password\n',

                'user create willbedeleteduser --first soon --last deleted\n',
                'user delete willbedeleteduser\n'

                'user create nopassworduser --first no --last password --no-password\n',

                'user create removedpassworduser --first removed --last password\n'
                'user modify removedpassworduser --remove-password\n',
            ])

        test_record_url = 'file://' + test_record

>       result = click_run(directory, ['import', test_record_url])

src/test_import_export.py:65:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/appliance_cli/testing_utils.py:9: in click_run
    result = runner.invoke(cli, args, catch_exceptions=False, **kwargs)
venv/lib/python3.5/site-packages/click/testing.py:279: in invoke
    prog_name=self.get_default_prog_name(cli), **extra)
venv/lib/python3.5/site-packages/click/core.py:697: in main
    rv = self.invoke(ctx)
src/directory.py:29: in invoke
    super().invoke(ctx)
venv/lib/python3.5/site-packages/click/core.py:1066: in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
venv/lib/python3.5/site-packages/click/core.py:895: in invoke
    return ctx.invoke(self.callback, **ctx.params)
venv/lib/python3.5/site-packages/click/core.py:535: in invoke
    return callback(*args, **kwargs)
src/import_export.py:55: in import_
    _output_imported_user_passwords()
src/import_export.py:87: in _output_imported_user_passwords
    _export_imported_user_passwords(sorted_passwords_data)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

passwords_data = [['anotheruser', 'anotheruser_user-mod_password'], ['newuser', 'newuser_user-add_password']]

    def _export_imported_user_passwords(passwords_data):
        unix_timestamp = math.floor(time.time())
        export_filename = 'passwords-{}.tsv'.format(unix_timestamp)
        export_filepath = os.path.join(CONFIG.EXPORT_DIR, export_filename)

>       with open(export_filepath, 'w', newline='') as export_file:
E       FileNotFoundError: [Errno 2] No such file or directory: '/var/www/html/secure/passwords-1529681686.tsv'

src/import_export.py:106: FileNotFoundError
================================================= 1 failed, 33 passed in 0.57 seconds =================================================
make: *** [unit-test] Error 1
```

**Comment by ColonelPanicks** [comment]
Test passed when:

- Created `/var/www/html/secure`

- Created `/opt/cluster/etc/access.rc` containing:
```
cw_ACCESS_fqdn=infra01.testnet.alces.network
```


This issue will be properly resolved by refactoring the code not to require that file to be in clusterware but instead to exist in `/opt/directory/etc/config`.

**Comment by ste78** [comment]
fix it stupot 👏

**Comment by ColonelPanicks** [comment]
Fixed in `e39a058`

**Comment by ste78** [comment]
woo!
